src/engine/server.py

```diff
@@ -1,9 +1,22 @@
 from fastapi import FastAPI
+from fastapi.responses import JSONResponse
 
-
+# fastapiを実行
 app = FastAPI()
 
-# 適当なダミーのエンドポイント
-@app.post("/test/{id}")
-async def test(id: str,):
-    print(f"got request {id}")
+# BaseModelをインポート
+from src.engine.model import OnPageOpen, OnGoogleSearch
```

Comment:
今のこのimportは，リポジトリルートで実行する前提になっているけど，実行時にはsrc/engine/に移動している状態になるので，↓のようにしといてちょ!!
```suggestion
from model import OnPageOpen, OnGoogleSearch
```